backend/services/table_anomaly_service.py

```
@@ -0,0 +1,433 @@
+                "detected_at": datetime.utcnow().isoformat(),
+                "total_records": 0,
+                "duplicate_count": 0,
+                "null_count": 0,
+                "anomaly_count": 0,
+                "status": "normal",
+                "is_acknowledged": False,
+                "type": "table_anomaly_detection",
+                "tenant_id": tenant_id
+            }
+
+    async def get_anomaly_details(self, detection_id: str, tenant_id: str = "default") -> List[Dict[str, Any]]:
```

Ruff (0.13.1)

Unused method argument: `tenant_id`

(ARG002)